cmake/FindPythonModule.cmake

```diff
@@ -57,7 +59,6 @@ macro (find_python_module module)
                         OUTPUT_VARIABLE _${module}_version
                         ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-      set (_${module_upper}_VERSION_MATCH TRUE)
```

Comment:
What did you have in mind with this change ? Currently it makes the i386 build fail as numpy as the cmake logic does not see anymore that it's not available with python3 (we configure for python3 in the same location as the python2 build).

Reply:
Variables used in macros must always be set to a default value, otherwise the value set in a previous call is used.  Anyway I'll revert this change.

Reply:
Hmmm I also kept the wrong FPHSA call :-/